201-rbac-managedidentity-maps/azuredeploy.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "subscriptionId": {
+      "type": "string",
+      "defaultValue": "2498e3dd-307d-4f0d-bc19-bdc139ce0c30",
+      "metadata": {
+        "description": "Subscription ID GUID"
+      }
+    },
+     "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "location for the the resources to deploy."
+      }
+    },
+    "userAssignedIdentityName": {
+      "type": "string",
+      "metadata": {
+        "description": "the name of the Managed Identity resource."
+      }
+    },
+    "mapsAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "the name of the Azure Map Account"
+      }
+    },
+    "builtInRoleType": {
+      "type": "string",
+      "defaultValue": "Azure Maps Data Reader",
+      "metadata": {
+        "description": "Input string for built in role type"
+      }
+    },
+    "guid": {
+      "type": "string",
+      "defaultValue": "916369fe-4b07-4b79-aa53-add064735d33",
```

Comment:
set this to

[guid(resourceGroup().id)]

Reply:
done